1.7/usage/tutorials/arangodb/index.md

```diff
@@ -46,45 +46,39 @@ Click the ArangoDB task to reveal the subtask that the framework has started:
 
 ![Tasks](img/tasks.png)
 
-Click “Open Service” to open the ArangoDB dashboard:
+Click “Open Service” to open the ArangoDB Webinterface.
 
-![Dashboard](img/dashboard.png)
-
-By default, ArangoDB will not expose itself to the outside. The IPs listed here are the internal IPs in the cluster. To access the nodes from the outside, we recommend using [sshuttle](https://github.com/sshuttle/sshuttle).
+![Dashboard](img/arangodb.png)
 
-The exact way to dig a tunnel using sshuttle varies from infrastructure to infrastructure. The following is an example for an AWS cluster:
+Congratulations! You now have ArangoDB running on DC/OS.
 
-```bash
-$ sshuttle --python /opt/mesosphere/bin/python3.4 -r core@54.171.143.132 10.0.0.0/8
-```
+## Using ArangoDB within the DC/OS cluster
 
-The IP may be extracted from the top left corner of the DC/OS web interface:
+Now that you have ArangoDB running you will likely want to fill it with some data and be able to use it as a data store for your applications.
 
-![Dashboard](img/ip.png)
+To do so you need to talk to the so called "Coordinators" of your ArangoDB cluster. As you can't be sure where they are located and due to the fact that they might move or change anytime throughout your cluster lifetime (task might fail, up- and downscaling and so forth) it is a bad idea to hardcode their IP addresses and ports in your applications.
```

Comment:
Consider revising to: `Now that ArangoDB is running you can fill it with some data and use it as a data store for your applications. To get started, talk to the "Coordinators" of your ArangoDB cluster. You should not hardcode the "Coordinators" IP addresses and ports in your applications because they can move or change at any time throughout your cluster lifetime (e.g. tasks might fail, up- and downscaling) .`